.github/workflows/notify-triager.yml

```diff
@@ -0,0 +1,69 @@
+name: Notify Triagers
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
```

Comment:
No need to run this workflow on master. It is only required for pull requests and issues.